packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -186,6 +186,9 @@ export interface IDetailsListProps extends React.Props<DetailsList> {
 
   /** Boolean value to indicate if the component should render in compact mode. Set to false by default */
   compact?: boolean;
+
+  /** Boolean value to indicate if the DetailsList Header should render in a Sticky component. Set to false by default */
```

Comment:
I think the best thing to do here would be to force the caller to use `onRenderHeader` and wrap the original `defaultRender` with a `Sticky` component. Don't force `DetailsList` to take a direct dependency on `Sticky`.